db/migrate/20160921184937_add_cancel_reason_detail_to_replication_transfer.rb

```diff
@@ -0,0 +1,5 @@
+class AddCancelReasonDetailToReplicationTransfer < ActiveRecord::Migration
+  def change
+    add_column :replication_transfers, :cancel_reason_detail, :text, null: true
+  end
+end
```

Comment:
Is `null: true` the default?

Do we need to add `limit:` option for the length of a text field?

Same questions apply to the other migration in this PR.